site/helm-upgrading-to-beta.md

```diff
@@ -37,10 +37,13 @@ since the new operator will ignore them.
 
 ### Using the flux chart
 
-The chart (from v0.5.0-beta, or from this git repo) provides the
+The chart (from v0.5.0, or from this git repo) provides the
 correct arguments to the operator; to upgrade, do
 
-   helm upgrade flux --reuse-values weaveworks/flux --version 0.5.0-beta
+```bash
```

Comment:
In most places we are using `sh`

https://github.com/weaveworks/flux/search?q=%60%60%60sh vs https://github.com/weaveworks/flux/search?q=```bash